ci-operator/jobs/openshift/os/openshift-os-release-4.16-presubmits.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
   openshift/os:
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
I'm just curious, what's the logic behind this boolean?

Reply:
Ok, NVM.
```    rerun_command: /test images```

Reply:
This part is automatically generated so I never looked at it :)